src/control/cmd/dmg/pool.go

```diff
@@ -37,15 +37,14 @@ import (
 )
 
 const (
-	msgSizeNoNumber = "size string doesn't specify a number"
-	msgSizeZeroScm  = "non-zero scm size is required"
-	maxNumSvcReps   = 13
+	maxNumSvcReps = 13
 )
 
 // PoolCmd is the struct representing the top-level pool subcommand.
 type PoolCmd struct {
 	Create       PoolCreateCmd       `command:"create" alias:"c" description:"Create a DAOS pool"`
 	Destroy      PoolDestroyCmd      `command:"destroy" alias:"d" description:"Destroy a DAOS pool"`
+	List         systemListPoolsCmd  `command:"list" alias:"l" description:"List DAOS pools"`
```

Comment:
A potential argument against (however no change is requested) - all of the other commands in this group refer to a single pool resource, whereas this one applies to all pools in the system. Understand the goal is to just create an alias that may be  more convenient to an admin than dmg system list-pools.

A future command that I believe will be added in this group is dmg pool list-containers so I would consider any potential collisions (from a human interface or any other code perspective) that could happen by adding that and by having this list alias. I guess alias "lc" for list-containers would work alongside alias "l" for (pool) list.

Reply:
From a UX perspective, not having this alias here reduces the tool's discoverability. I have heard from a number of people (as recently as this morning, even) that they didn't realize that there was a way to list pools using dmg...